### ISSUE TITLE
ubuntu (and linuxmint) seem to have md5sum, not md5 installed by default

### ISSUE THREAD

**Issue by rliebling** [gliderlabs/glidergun#4]
so, depending on md5 to compute a checksum (when installed dependencies) is troublesome.  When i alias md5sum as md5, it computes the checksum but the md5sum output includes the filename (or a `-` for stdin):

```
$ cat .gun/tmp/jq | md5sum
89c7bb6138fa6a5c989aca6b71586acc  -
```

not sure if macosx has md5sum installed (so you could just always use that) or if you need to differentiate the cases.

**Comment by rliebling** [comment]
this resolved the issue for me (temporarily anyway):

```
$ cat ~/bin/md5
#!/bin/bash

md5sum $* | cut -d' ' -f1
```

**Comment by progrium** [comment]
That's a great catch. I thought I fixed this and then I realized it was for the tests on our index:
https://github.com/gliderlabs/glidergun-rack/blob/master/tests#L3

We can pretty easily export a custom md5 command from Go with glidergun that will work the same across all platforms. Thanks for banging on glidergun. :)

**Comment by progrium** [comment]
Also fixed in 0.0.5
